tests/plugins/imagebase/features/manual/caption.html

```diff
@@ -0,0 +1,34 @@
+<div id="classic">
```

Comment:
Please, add inline editors. You  have added `floatingspace` plugin in `.md` file but you did not use it with any editor.